examples/basic/kernel/Cargo.toml

```
@@ -0,0 +1,9 @@
+[package]
+name = "kernel"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+bootloader_api = "0.11.12"
+uart_16550 = "0.4.0"
+x86_64 = { version = "0.15.2", features = ["instructions"] }
```

```suggestion
x86_64 = "0.15.2"
```
`instructions` is enabled by default.